frameioclient/client.py

```diff
@@ -322,21 +322,22 @@ def upload(self, asset, file):
     """
     uploader = FrameioUploader(asset, file)
     uploader.upload()
-  
-  def download(self, asset, download_folder):
-    """
-    Download an asset. The method will exit once the file is downloaded.
 
-    :Args:
-      asset (object): The asset object.
-      download_folder (path): The location to download the file to.
+  def download(self, asset, download_folder, replace=True):
+      """
+      Download an asset. The method will exit once the file is downloaded.
 
-      Example::
+      :Args:
+        asset (object): The asset object.
+        download_folder (path): The location to download the file to.
+        replace (bool): Replace the file if it exists.
```

Comment:
I do really like this feature :). Ran into it a ton of times in my testing of the multi-threaded downloader...